src/openrct2/drawing/Drawing.h

```diff
@@ -13,6 +13,7 @@
 #include "../common.h"
 #include "../interface/Colour.h"
 #include "../interface/ZoomLevel.hpp"
+#include "../world/Location.hpp"
```

Comment:
Forward declare the `ScreenRect` below (line ~21) so we don't have to add this include here, please